### PR TITLE
Add citation file

### LINKED ISSUE #378: Set up Zenodo linking and citation information
### Summary

We should add a citation entry to the repository, and start deploying to Zenodo so people can cite the releases.
https://citation-file-format.github.io/
https://docs.github.com/en/repositories/archiving-a-github-repository/referencing-and-citing-content

### Requirements

  - [x] Link Zenodo to the `imap_processing` repository
  - [x] Add a citation.cff file with the proper metadata to the `imap_processing` repository

### Related tickets

_No response_

### PR DESCRIPTION
**Pull request by vmartinez-cu**
# Change Summary

## Overview
Added a citation.cff file to the repository to provide the information needed to properly cite this software and support integration with Zenodo. Zenodo will use this file to collect details for publishing a DOI. Note that this file will need to be updated with a DOI once Zenodo automatically creates one with the next release. I've placed the file at the root of the repo so that it can easily be found.

## New Files
- citation.cff
   - Includes information needed to properly cite this repository

### CODE REVIEW

File: CITATION.cff
Comment (greglucas) [inline CITATION.cff]: I would remove data level discussion as that is somewhat confusing/contentious in how we would describe that.

```suggestion
  expand our understanding of space. The Science Operations
  Center (SOC) is located at the Laboratory of Atmospheric and Space Physics (LASP)
  and is responsible for the data processing on the IMAP mission.
```

File: CITATION.cff
Comment (greglucas) [inline CITATION.cff]: Are we supposed to put the word developers or something else to indicate it is people behind this, or is center/group the canonical way to represent that?
```suggestion
  - name: IMAP Science Operations Center Developers
```

**Review by greglucas — APPROVED** [review]
Looks great to me, only a minor nit on the final sentence of the abstract and one small comment below that can be ignored.

We are technically the "Science Data Center" which is underneath the "Science Operations Center" at LASP. Do we want to make that distinction here and say IMAP SDC? I suppose that is confusing because the organization on GitHub is the "Science-Operations-Center", so maybe just leave it as you currently have it 🤷 .

File: CITATION.cff
Comment (bourque) [inline CITATION.cff:25]: Would it make sense to add some science-y words to this list? (e.g. `Heliophysics`). I'm not sure what these keywords are used, is it some sort of search functionality on GitHub or Zenodo?

**Review by bourque — APPROVED** [review]
Looks great!

File: CITATION.cff
Comment (vmartinez-cu) [reply CITATION.cff]: That's a great point. I think we can be more clear and use "Science Data Center" here instead. To reduce confusion, this repo's readme should probably be updated as well to mention the SDC and it's role under the SOC.

File: CITATION.cff
Comment (vmartinez-cu) [reply CITATION.cff:25]: Yeah I think adding some additional keywords would be good. I believe Zenodo will use these for its search functionality.

File: CITATION.cff
Comment (vmartinez-cu) [reply CITATION.cff]: Updated author to IMAP Science Data Center Developers and added a blurb about the SDC in the abstract

File: CITATION.cff
Comment (tech3371) [reply CITATION.cff:25]: keywords addition suggestion:
algorithm development
Energetic Neutral Atom (ENA)
Interstellar Mapping and Acceleration Probe (IMAP) - spelled out version?

**Review by tech3371 — APPROVED** [review]
looks great!

File: CITATION.cff
Comment (vmartinez-cu) [reply CITATION.cff:25]: Great suggestions! Additional keywords have been added

**Comment by tech3371** [comment]
btw, you don't have to wait for approval from everyone. If you feel like you got plenty feedback or in some cases, if you got feedback from person that you require input from, you can merge PR if all the checks passed.

**Comment by vmartinez-cu** [comment]
Okay great, I wasn't sure if I should merge my own PRs. Thanks for the feedback and reviews!

**Pull request merged**
